OrbitCore/Params.cpp

```diff
@@ -67,12 +37,12 @@ bool Params::Save() {
     return false;
   }
 
-  try {
-    cereal::XMLOutputArchive archive(file);
-    archive(cereal::make_nvp("Params", *this));
+  google::protobuf::io::OstreamOutputStream output_stream(&file);
+  if (google::protobuf::TextFormat::Print(config, &output_stream)) {
```

Comment:
nitpick:

if (! .. ) {
   ...
   return false;
}

return true;

Reply:
btw what is output format here?

Reply:
The sample of `config.textpb` file:
```
track_context_switches: true
track_sampling_events: true
unreal_support: true
start_paused: true
find_file_and_line_info: true
font_size: 14